esp/bindings/http/platform/httptransport.ipp

```diff
@@ -438,6 +443,48 @@ public:
     virtual bool decompressContent(StringBuffer* originalContent, int compressType);
 };
 
+class CFlushingHttpResponseBuffer : public CInterfaceOf<IXmlStreamFlusher>
+{
```

Comment:
Is this only used by ws_workunitHelpers?  Could you move this class and all the following helpers:

```
  unsigned sendComponentLogContent(IEspContext* context, IRemoteLogAccessStream* logreader, IXmlStreamFlusher* flusher, const WUComponentLogOptions& options);
    void sendComponentLogCSV(IEspContext* context, IRemoteLogAccessStream* logreader, IXmlStreamFlusher* flusher, const WUComponentLogOptions& options);
    void sendComponentLogJSON(IEspContext* context, IRemoteLogAccessStream* logreader, IXmlStreamFlusher* flusher, const WUComponentLogOptions& options);
    void sendComponentLogXML(IEspContext* context, IRemoteLogAccessStream* logreader, IXmlStreamFlusher* flusher, const WUComponentLogOptions& options);
```
Directly into ws_workunithelpers.cpp, then you wouldn't need all of the CMakeLists.txt changes in all those files.

Reply:
Unless you think this may be used in other places?

Reply:
This may be used in other places. But, do not have to be in httptransport.ipp. I may add this class into new file (esp/bindings/http/platform/httpresponsebuffer.ipp?).

Reply:
Yes, something like that.  In the end the extra dependencies wherever httptransport.ipp is used should go away.  Then someplace else uses httpresponsebuffer.ipp any dependencies will only be needed where used.